setup.py

```diff
@@ -1,17 +1,15 @@
 from setuptools import setup
 
-import pybrake
 
-
-def readme():
-  with open('README.md') as f:
+def content_of(file_name):
+  with open(file_name) as f:
     return f.read()
 
 
 setup(name='pybrake',
-      version=pybrake.__version__,
+      version=content_of('pybrake/version.py').strip(),
```

Comment:
Will not this return `version = "0.4.0-beta.4"` instead of `0.4.0-beta.4`?

Reply:
Ah, you're right. Well, `pybrake` requires `tdigest` and that mean you cannot import pybrake from setup.py. I'll reimplement version handling and update the PR.

Reply:
It's been fixed.